plugins/test/fu-plugin-test.c

```diff
@@ -106,6 +108,13 @@ fu_plugin_update (FuPlugin *plugin,
 		  FwupdInstallFlags flags,
 		  GError **error)
 {
+	if (g_strcmp0 (g_getenv ("FWUPD_PLUGIN_TEST"), "fail") == 0) {
+		g_set_error_literal (error,
+				     FWUPD_ERROR,
+				     FWUPD_ERROR_NOT_SUPPORTED,
+				     "device was not in supported mode");
```

Comment:
Small initial d in device vs capital D in Device in the hunk above :P

Reply:
I'm genuinely not sure if GError messages are supposed to start with a capital.